template/.eslintrc.js

```diff
@@ -44,6 +44,8 @@ module.exports = {
       'js': 'never',
       'vue': 'never'
     }],
+    // allow vuex
+    'no-param-reassign': ["error", { 'props': false }],
```

Comment:
I agree. However, it's very unfriendly to throw errors when using official library Vuex.